Source/svg/SVGParser.swift

```diff
@@ -608,6 +608,10 @@ open class SVGParser {
             let color = Color(val: defaultColor)
             return opacity != 1 ? color.with(a: opacity) : color
         }
+        if let systemColor = SVGConstants.systemColorList[colorString] {
+            let color = Color(val: systemColor)
```

Comment:
Why you didn't apply `opacity` to the color? It looks like it should be applied.

Reply:
yeah, that should be there